stores/ProgressStore.js

```diff
@@ -32,7 +32,7 @@ export const useProgressStore = defineStore("progress", {
   },
   actions: {
     init() {
-      // Necessary for SSR
+      /** Necessary for SSR */
```

Comment:
Das ist kein JSDoc-Kommentar, der muss so wie JavaDoc etwas kommentieren
```suggestion
      // Necessary for SSR
```